### ISSUE TITLE
Title screen CTD

### ISSUE THREAD

**Issue by MrCool92** [OpenRCT2/OpenRCT2#17054]
### Operating System

Windows 10, 64-bit

### OpenRCT2 build

OpenRCT2, v0.3.5.1 (61c67af) provided by GitHub

### Describe the issue

I was browsing the options menu on the title screen.

I had only RTC2 installed at the time.

### Steps to reproduce

Not really sure... Can't reproduce it.

### Attachments

[openrct2.exe.6412.zip](https://github.com/OpenRCT2/OpenRCT2/files/8554794/openrct2.exe.6412.zip)

**Comment by tupaschoal** [comment]
Were you prompted to submit a crash report (and did you)?

**Comment by Broxzier** [comment]
@tupaschoal Yes, on Discord. OP asked where they could upload a crash dump, and was told to create an issue here.

**Comment by Gymnasiast** [comment]
It's been three years and I have no way of investigating this report. I hope it’s been solved in the meantime.
